deepchem/models/torch_models/antibody_modeling.py

```diff
@@ -1,128 +1,149 @@
-    >>> # Expected Output
-    >>> # optimized_sequences[0]
-    >>> # (0,'Q','QSETQDPAVSVALGQTVRITCQGDSLRNYYASWYQQKPRQAPVLVFYGKNNRPSGIPDRFSGSSSGNTASLTISGAQAEDEADYYCNSRDSSSNHLVFGGGTKLTVLSQ',0.7314766049385071)
-    """
+
+   DeepAbLLM is a wrapper class that leverage large language model's (LLMs) learned sequence
+   co-dependencies to aid in the (re)-designing anitbody sequences. It supports the instantiation
+   of an arbitrary HuggingFace transformer-style model trained on Antibody sequences for the antibody
+   sequence redesign, extending an approach introduced in Hie et al's 2023 Nature Biotech paper [1].
+
+
+   This means the functionality of DeepAbLLM is model architecture-agnostic.
+
+
+   Currently supports a variety of HuggingFace Protein and Antibody Specific Language Models, including:
+
+
+   [2] ProtTrans Models (ProtBERT, ProtT5, etc.)
+   [3] AbLang
+   [4] IgBERT/IgT5
+   [5] ESM1b
+   [6] ESM1v
+   [7] ESM-2
+
+
+   The model uses single amino acid tokenization to create input tokens for the models from the
+   antibody sequences. While most protein models expect spaces in the protein sequences:
+       "T H I S I S A P R O T E I N S E Q U E N C E"
+   the ESM class of models does not and expects strings of the following format:
+       "THISISAPROTEINSEQUENCE"
+   Both tokenization schemes are supported by DeepAbLLM by setting the is_esm_variant flag to the
+   appropriate value.
+
+
+   The model supports general pretraining via masked language modeling, domain-adaptive pretraining [8]
+   - an additional pretraining step applied to general purpose protein language models for antibody sequences,
+   and the finetuning of pre-trained models for regression/classification. To pretrain via masked language
+   modeling task, use task = `mlm`, 'regression', or `classification` during initialization.
+
+
+   Attributes
+   ----------
+   task: str
+       The task the HuggingFaceModel is performing. Default: 'mlm'.
+   model_path: str
+       The huggingface model path of the pLM.
+   n_tasks: int
+       Number of tasks for a given model. Default: 1
+   is_esm_variant: bool
+       Boolean flag to indicate the tokenization scheme.
+
+
+   Methods
+   -------
+   __init__(task, model_path, n_tasks)
+       Initialize an DeepAbLLM with specified information.
+   _mask_seq_pos(position, idx)
+       Mask a sequence at a particular index.
+   redesign_residue(sequence, residue_index: int, top_k, verbose)
+       Mask and unmaks a single residue of a sequence at given index, returning
+       top-k plausible amino acid substitutions.
+   _optimize_residue_pos(sequence, residue_index, verbose, threshold)
+       "Optimizes" a residue position by redesigning it and returning
+       only the tokens that rank higher than the original token.
+   redesign_sequence(sequence)
+       Returns optimized sequences over each residue position with better
+       scores than the original sequence.
+
```

Comment:
Possibly an editor issue? Lots of introduced white space

Reply:
Fixed the white space issue.